### PR TITLE
Add header to workload list command output

### PR DESCRIPTION
**Pull request by sfoslund**

### CODE REVIEW

**Review by joeloff — APPROVED** [review]

File: src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx [hunk 1]
```diff
@@ -129,4 +129,7 @@
   <data name="WorkloadListFooter" xml:space="preserve">
     <value>Use `dotnet workload search` to find additional workloads to install.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="WorkloadListHeader" xml:space="preserve">
+    <value>This command lists only workloads that were installed via `dotnet workload install` in this version of the SDK and not those that were installed via Visual Studio.</value>
```
Comment (Forgind) [inline src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx:133]:
```suggestion
    <value>This command lists only workloads that were installed via `dotnet workload install` in this version of the SDK and not those that were installed via Visual Studio Installer.</value>
```
?

**Review by Forgind — APPROVED** [review]

File: src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx (hunk 1 above)
Comment (sfoslund) [reply src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx:133]: I'm going to leave this as is for now as Kathleen/PMs came up with the wording and have already approved offline.

Comment (KathleenDollard) [reply src/Cli/dotnet/commands/dotnet-workload/list/LocalizableStrings.resx:133]: <value>This command lists only workloads that were installed via `dotnet workload install` for this version of the SDK. It does not list workloads that were installed via Visual Studio Installer.</value>

I found the long sentence hard to read, and think Nathan's addition of the word installer is good.

**Pull request merged**